ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -439,7 +440,7 @@ def ndmg_dwi_worker(
 
         # Align DWI volumes to Atlas
         print("Aligning volumes...")
-        reg = mgr.dmri_reg_old(dwi_prep, gtab, t1w, atlas, aligned_dwi, namer, clean)
+        reg = mgr.dmri_reg_old(dwi_prep, gtab, t1w, mask, aligned_dwi, namer, clean)
```

Comment:
whoa, the atlas was being passed instead of the mask?

Reply:
```
class dmri_reg_old(object):
    def __init__(self, dwi, gtab, t1w, atlas, aligned_dwi, namer, clean=False)
```
It looks like the argument after t1w *should* be the atlas. Why are we passing a mask now? 
(Maybe I am just misremembering what the variables are?)

Reply:
the variable 'atlas' for the mni-space pipeline should be a file path, but in the new ndmg pipeline, it is a string (e.g. 'desikan'). So, mask is just the path to the template which is used as reference for the registration.

Reply:
Cool